integration-tests/src/test/resources/features/AddressAPIHappyPath.feature

```diff
@@ -69,3 +69,35 @@ Feature: Address API happy path test
       | 197                        | SW1A 2AA     |
       | 23                         | CA14 5PH     |
       | 1000                       | S62 5AB      |
+
+  @international_address_api_happy
+  Scenario: Temporary Country Code Only International Address API journey
+    Given user has the test-identity 197 in the form of a signed JWT string
+
+    # Session
+    When user sends a POST request to session end point
+    Then user gets a session-id
+
+    # TXMA event
+    Then TXMA event is added to the SQS queue not containing device information header
+
+    # Postcode lookup
+    When the user performs a postcode lookup for post code "SW1A 2AA"
+    Then user receives a list of addresses containing "SW1A 2AA"
+
+    # Address
+    When the user selects international address
+    Then the address is saved successfully
+
+    # Authorization
+    When user sends a GET request to authorization end point
+    And a valid authorization code is returned in the response
+
+    # Access token
+    When user sends a POST request to token end point
+    And a valid access token code is returned in the response
+
+    # Credential Issued
+    When user sends a POST request to Credential Issue end point with a valid access token
+    And a valid JWT is returned in the response
+    And 5 events are deleted from the audit events SQS queue
```

Comment:
FYI @MarieseMikely as this is related to the work you're currently doing - might want to rebase after this has been merged if you're changing any of the features